lib/screens/profile/profile_screen.dart

```diff
@@ -0,0 +1,222 @@
+                      "Payments methods",
+                      style: TextStyle(
+                        color: AppColors.black,
+                        fontWeight: FontWeight.bold,
+                      ),
+                    ),
+                    subtitle: Text(
+                      "visa **34",
+                      style: TextStyle(
+                          color: AppColors.lightGray,
+                          fontWeight: FontWeight.bold),
+                    ),
+                    trailing: Icon(Icons.chevron_right),
+                  ),
+                  onTap: () {},
+                ),
+                Divider(),
+                InkWell(
+                  child: ListTile(
+                   
+                    title: Text(
+                      "Promocodes",
+                      style: TextStyle(
+                        color: AppColors.black,
+                        fontWeight: FontWeight.bold,
+                      ),
+                    ),
+                    subtitle: Text(
+                      "You hve special promocodes",
+                      style: TextStyle(
+                          color: AppColors.lightGray,
+                          fontWeight: FontWeight.bold),
+                    ),
+                    trailing: Icon(Icons.chevron_right),
+                  ),
+                  onTap: () {},
+                ),
+                Divider(),
+                InkWell(
+                  child: ListTile(
+                    
+                    title: Text(
+                      "My reviews",
+                      style: TextStyle(
+                        color: AppColors.black,
+                        fontWeight: FontWeight.bold,
+                      ),
+                    ),
+                    subtitle: Text(
+                      "review for 4 items",
+                      style: TextStyle(
+                          color: AppColors.lightGray,
+                          fontWeight: FontWeight.bold),
+                    ),
+                    trailing: Icon(Icons.chevron_right),
+                  ),
+                  onTap: () {},
+                ),
+                Divider(),
+                InkWell(
+                  child: ListTile(
+                    
+                    title: Text(
+                      "Settings",
+                      style: TextStyle(
+                        color: AppColors.black,
+                        fontWeight: FontWeight.bold,
+                      ),
+                    ),
+                    subtitle: Text(
+                      "Notification, Password",
+                      style: TextStyle(
+                          color: AppColors.lightGray,
+                          fontWeight: FontWeight.bold),
+                    ),
+                    trailing: Icon(Icons.chevron_right),
+                  ),
+                  onTap: () {},
+                )
+              ],
+            )
+          ],
+        ));
```

Comment:
could you please refactor the code? I suggest:

1. create a list of options with title, subtitle, icon and onTap (command)
2. create a method to build the options widgets
3. create a method to build the divider